File: Core/Entities/LoanProduct.cs
namespace Core.Entities
{
    public class LoanProduct
    {
        public int LoanProductID { get; set; }
        public string ProductName { get; set; }
        public string Description { get; set; }
        public decimal InterestRate { get; set; }
        public decimal MaxLoanAmount { get; set; }
        public int MinCreditScore { get; set; }
        public int MinTermMonths { get; set; }
        public int MaxTermMonths { get; set; }
        public bool IsActive { get; set; }
    }
}

File: Angular/loan-management-system/src/app/models/loan-product.ts
export interface LoanProduct {
  loanProductId: number;
  productName: string;
  description: string;
  interestRate: number;
  maxLoanAmount: number;
  minCreditScore: number;
  minTermMonths: number;
  maxTermMonths: number;
  isActive: boolean;
}
